protostar/testing/test_config_test.py

```diff
@@ -8,6 +8,9 @@
     [
         (TestMode.STANDARD, TestMode.FUZZ, True),
         (TestMode.FUZZ, TestMode.STANDARD, False),
+        (TestMode.STANDARD, TestMode.PARAMETERIZED, True),
+        (TestMode.PARAMETERIZED, TestMode.FUZZ, True),
+        (TestMode.FUZZ, TestMode.PARAMETERIZED, True),
```

Comment:
please add a test which checks that 
```python
TestMode.FUZZ.convert_to(TestMode.PARAMETERIZED) is TestMode.FUZZ
```

Reply:
[forbid converting fuzz test to parameterized test](https://github.com/software-mansion/protostar/pull/947/commits/7026c691fe9502aff8de957e10b3975f253f285c)